### ISSUE TITLE
Remove `templates` directory

### ISSUE THREAD

**Issue by CaporalDead** [JackMandrake/gwith#2]
It seems that this directory nor the template are used in your app.

**Comment by MilanForrat** [comment]
Yes, that's true, we tried using that directory for some tests regarding EasyAdmin and password reset, we ended up using them in our latest repository (25/09/2020). Thanks for that review ;)
